src/limit-line-length/limit-line-length.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import addStyleToPage from '../add-style'
+
+function createCssRules(lineLengthLimit, isStickyHeaderEnabled) {
+    let filler = ''
+    for (let i = 0; i < lineLengthLimit; i++) {
+        filler += '#'
+    }
+
+    let cssRules = `
+        .refract-content-container > .udiff-line:first-child .source::before,
```

Comment:
I needed to reference only one of the `.udiff-line` elements (didn't care which one) inside `.refract-content-container`.
As far as i know this is the best way to do it (:first-of-type won't help here, nor :first-child/:last-child)